packages/onboarding/package.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "@automattic/onboarding",
+	"version": "1.0.0",
+	"description": "Onboarding components for WordPress.com",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/onboarding"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"types": "dist/types",
+	"scripts": {
+		"clean": "check-npm-client && npx rimraf dist ../../.tsc-cache/packages__onboarding*",
+		"prepare": "check-npm-client && tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json && copy-assets && npx copyfiles ./styles/** dist",
+		"prepublish": "check-npm-client && yarn run clean",
+		"watch": "check-npm-client && tsc --project ./tsconfig.json --watch"
+	},
+	"dependencies": {
+		"@automattic/react-i18n": "^1.0.0-alpha.0",
+		"@wordpress/components": "^9.6.0",
+		"classnames": "^2.2.6"
+	},
+	"devDependencies": {
+		"@wordpress/base-styles": "^1.8.0",
```

Comment:
Needs `@automattic/typography` too?

Reply:
Good catch! Added to onboarding & plans-grid package. https://github.com/Automattic/wp-calypso/pull/44077/commits/d4633a6b81ab0376d4ef0d894d9cba3887d8fc02

Reply:
Shouldn't this be added also to FSE deps? I see [@wordpress/base-styles](https://github.com/Automattic/wp-calypso/blob/d4633a6b81ab0376d4ef0d894d9cba3887d8fc02/apps/full-site-editing/package.json#L102) there

Reply:
Done.